ush/python_utils/config_parser.py

```diff
@@ -604,7 +604,15 @@ def load_config_file(file_name, return_string=0):
         return load_yaml_config(file_name)
     if ext == "xml":
         return load_xml_config(file_name, return_string)
-    return None
+    else:
```

Comment:
This `else` block isn't strictly necessary. The error raising is fantastic and useful. If the flow gets through all the if statements, then this code will be executed without the need for `else:`.

Reply:
Done, also did some more simplification since I recently learned that the `+=` operator works to concatenate strings also!